beacon-chain/sync/rpc_status.go

```diff
@@ -36,7 +38,14 @@ func (r *RegularSync) maintainPeerStatuses() {
 					}
 				}
 			}
-
+			_, highestEpoch := r.p2p.Peers().HighestFinalizedPeer()
+			if highestEpoch > r.chain.FinalizedCheckpt().Epoch {
```

Comment:
Can we make it that our head slot is behind farther than the highest finalized epoch from peers? This would be a signal that we are very far behind and should resync.

Reply:
sounds good, I can change it to that